src/modules/Sudo.rb

```diff
@@ -566,7 +589,16 @@ def PollAbort
     def Read
       return false if !Confirm.MustBeRoot
 
-      Report.Error(Message.CannotReadCurrentSettings) if !ReadSudoSettings2()
+      begin
+        Report.Error(Message.CannotReadCurrentSettings) if !ReadSudoSettings2()
+      rescue UnsupportedSudoConfig => e
+        msg = _("Unsupported configuration found. YaST2 exits now to prevent breaking system.")
```

Comment:
What about `YaST will not exit to prevent from breaking the system.`?

Reply:
fine for me